### PR TITLE
Integrate with TravisCI

### PR DESCRIPTION
**Pull request by JuannFerrari**
### What changes does this PR?

- This PR configure TravisCI, add badges to `README.md` and adds basic Rspec Structure.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  global:
+    - CC_TEST_REPORTER_ID=a3068b786ab61159b292bd4bae496bd03f98893d2016b5a73e06c185550517b2
+
+before_script:
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+  - chmod +x ./cc-test-reporter
+  - ./cc-test-reporter before-build
+
+script:
+  - bundle exec rake code_analysis
```
Comment (santib) [inline .travis.yml:24]: question, do we have this task in this repo? if not we might want to add it

File: activeadmin-async_exporter.gemspec [hunk 2]
```diff
@@ -19,9 +19,12 @@ Gem::Specification.new do |s|
 
   # Dependencies
   s.add_runtime_dependency 'activeadmin', '>= 1.0.0.pre1'
-
-  s.add_dependency 'rails', '>= 1.0.0'
   s.add_dependency 'aws-sdk-s3', '~> 1'
+  s.add_dependency 'rails', '>= 1.0.0'
 
+  # Development dependencies
+  s.add_development_dependency 'reek', '~> 5.5.0'
```
Comment (santib) [inline activeadmin-async_exporter.gemspec]: If we add `reek` let's add the config yml, otherwise we can remove reek

**Review by santib — APPROVED** [review]
👏

File: .travis.yml (hunk 1 above)
Comment (JuannFerrari) [reply .travis.yml:24]: Added on last commit 🙌

**Review by jpascual1994 — APPROVED** [review]
🚀

**Pull request merged**